### PR TITLE
Add some camera adjustments

### PR DESCRIPTION
**Pull request by matthewturk**
I'm going to add more to this, but for now I've got a few new buttons for the camera.

### CODE REVIEW

**Review by matthewturk — COMMENTED** [review]
I've added the ability to modify the scroll wheel delta as well.  I think this is ready to go.

File: yt_idv/simple_gui.py [hunk 1]
```diff
@@ -101,10 +109,29 @@ def render_camera(self, scene):
                 scene.camera.position = np.array([0.499, 0.499, 0.499])
                 scene.camera.focus = np.array([0.5, 0.5, 0.5])
                 changed = True
+            imgui.same_line()
```
Comment (chrishavlin) [inline yt_idv/simple_gui.py:112]: oh! didn't know this imgui command, that's useful.

File: yt_idv/simple_gui.py [hunk 2]
```diff
@@ -72,6 +74,12 @@ def render_camera(self, scene):
         if not imgui.tree_node("Camera"):
             return
         changed = False
+        _, scroll_delta = imgui.slider_float(
+            "Log Scroll Delta", math.log10(scene.camera.scroll_delta), -3.0, 0.0
+        )
```
Comment (chrishavlin) [inline yt_idv/simple_gui.py:79]: you have to drag the menu a pretty far ways to read the full "Log Scroll Data" label: 

Screenshot from 2024-12-05 13-47-56

Making the slider narrower would make that easier to read. e.g.,:

```suggestion
        imgui.push_item_width(100)
        _, scroll_delta = imgui.slider_float(
            "Log Scroll Delta", math.log10(scene.camera.scroll_delta), -3.0, 0.0
        )
        imgui.pop_item_width()
```

Comment (matthewturk) [reply yt_idv/simple_gui.py:79]: oh jeez, wow.  Should I put them on different lines?  Or use this?

Comment (chrishavlin) [reply yt_idv/simple_gui.py:79]: Oh, different lines would work too. If you use the push/pop item width, it'd look like 

Screenshot from 2024-12-05 16-04-07

I don't have a strong preference on this versus splitting the label and slider to different lines, up to you!

**Comment by matthewturk** [comment]
Separate line mightb e nicer.  I'll try both and update this!

File: yt_idv/simple_gui.py (hunk 2 above)
Comment (matthewturk) [reply yt_idv/simple_gui.py]: I used multiple lines after trying both.

**Review by chrishavlin — APPROVED** [review]

**Pull request merged**